src/js/render_wordle.js

```diff
@@ -29,9 +29,7 @@ inverted_wordles.extractAnswers = function (answersFile) {
         counts[answer] = counts[answer] ? counts[answer] + 1 : 1;
     };
     Object.values(answersFile).forEach(function (oneAnswerRec) {
-        oneAnswerRec.answers.forEach(function (answer) {
-            answer.toLowerCase().split(" ").forEach(storeAnswer);
-        });
+        oneAnswerRec.answers.forEach(storeAnswer);
```

Comment:
We should probably still retain the "toLowerCase" action which was requested by @michelled - also a trim() probably wouldn't go amiss

Reply:
@amb26, the comment has been addressed. Ready for another review.